crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__version_1.snap

```diff
@@ -33,7 +33,8 @@ max_num_event_emit: 256
 max_num_new_move_object_ids: 2048
 max_num_deleted_move_object_ids: 2048
 max_num_transfered_move_object_ids: 2048
-max_event_emit_size: 262144
+max_event_emit_size: 256000
+max_move_vector_len: 262144
```

Comment:
seems a bit odd of a number to me? Any particular reason for this number?

Reply:
256 * 1024 = 2^18.
But I agree it does look weird. We can always change as we go though.

Reply:
maybe we should write those more like multiple of 1K